citations-tool/tool/src/webapp/css/citations.css

```diff
@@ -176,7 +176,7 @@ ol li.h1Editor {
 ol li div h3, ol li div h2, ol li div h1 {
 	margin: 0em 0 0;
 }
-ol.serialization li, ol.default li {
+ol.serialization li div.h1Editor, ol.serialization li div.h2Editor, ol.serialization li div.h3Editor, ol.serialization ol.h4NestedLevel li div  {
```

Comment:
Would it be clearer to just include in the HTML a class to mark a section as expandable/collpaseable?